transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/Persistor.java

```diff
@@ -20,7 +20,10 @@ public interface Persistor {
    * @return The persistor.
    */
   static DefaultPersistor forDialect(Dialect dialect) {
-    return DefaultPersistor.builder().dialect(dialect).build();
+    return DefaultPersistor.builder()
+        .dialect(dialect)
+        .sequenceGenerator(DefaultSequenceGenerator.builder().dialect(dialect).build())
```

Comment:
This shouldn't be necessary (see comments on `DefaultPersistor`)

Reply:
fixed